app/templates/activate.hbs

```diff
@@ -1,12 +0,0 @@
-{{page-title "Activate"}}
-<div class="auth-main-container">
-  <AuthAssets></AuthAssets>
-  <div class="activate-container">
-    <div class="activate-main">
```

Comment:
can you remove `activate-container` & `activate-main` style definitions if not used

Reply:
removed